controllers/imageupdateautomation_controller.go

```diff
@@ -199,12 +199,17 @@ func (r *ImageUpdateAutomationReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 			r.event(auto, events.EventSeverityInfo, "no updates made")
 			log.V(debug).Info("no changes made in working directory; no commit")
 			statusMessage = "no updates made"
+			if lastCommit, lastTime := auto.Status.LastPushCommit, auto.Status.LastPushTime; lastCommit != "" {
+				statusMessage = fmt.Sprintf("%s; last commit %s at %s", statusMessage, lastCommit, lastTime.Format(time.RFC3339))
```

Comment:
Can we use the short sha here?

Reply:
Yeahp; I'll just shorten it blindly (i.e., without ensuring it's still unique), since this is informational.